style.css

```diff
@@ -0,0 +1,190 @@
+html, body {
+  height: 100%;
+}
+body {
+  display: flex;
+  flex-direction: column;
+}
+.content {
+  flex: 1 0 auto;
+}
+
+.site-header {
+  background-color: #f2b4f4;
+}
+
+.site-main {
+background-color: #F4F4F4;
+}
+
+.site-footer {
+  background-color: #072BCE;
+  color: #ffffff;
```

Comment:
I have seen this inconsistency many places. Either go for the upper case letters or lower case letters.
so:
```css
#072bce
#ffffff
```
or
```css
#072BCE
#FFFFFF
```

Reply:
Done.